frontend/views/components/GroupSettings/InviteLinks.vue

```diff
@@ -0,0 +1,287 @@
+<template lang="pug">
+page-section.c-section(:title='L("Invite links")')
+  template(#cta='')
+    .select-wrapper.c-select-wrapper(:class='ephemeral.selectbox.focused ? "focused" : ""')
+      select.button.is-small.is-outlined.c-select(
+        ref='select'
+        v-model='ephemeral.selectbox.selectedOption'
+        @change='unfocusSelect'
+      )
+        option(value='Active') {{ L('Active links') }}
+        option(value='All') {{ L('All links') }}
+      span.c-rect
+
+  i18n.has-text-1.c-invite-description(tag='p') Here's a list of all invite links you own
+
+  table.table.c-table(v-if='activeList.length')
+    thead
+      tr
+        i18n.c-name(tag='th') created for
+        i18n.c-invite-link(tag='th') invite link
+        i18n.c-state(tag='th') state
+        th.c-action(
+          :aria-label='L("action")'
+        )
+    tbody
+      tr(
+        v-for='(item, index) in activeList'
+        :key='index'
+      )
+        td.c-name
+          | {{ item.name }}
+          tooltip.c-name-tooltip(
+            direction='top'
+            :isTextCenter='true'
+            :text='L("This invite link was only available during the onboarding period.")'
+          )
+            button.is-icon-small(v-if='item.name === "Anyone"')
```

Comment:
This link needs to have a special property on it to check it, for example, you can add a `isAnyoneLink` property to just that link, set it to `true`, and check that here